### PR TITLE
Add Visual Studio Code instructions

### PR DESCRIPTION
**Pull request by JeroenDM**
Add instructions to README.md to setup clang formatting in Visual Studio Code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +81,16 @@ Configuration Clang format in Sublime:
 
 https://github.com/rosshemsley/SublimeClangFormat/blob/master/README.md
 
+### Usage with Visual Studio Code
+
+Install the [c/c++ extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools).
```
Comment (davetcoleman) [inline README.md]: C/C++ should be capitalized

**Review by davetcoleman — CHANGES_REQUESTED** [review]
love it!

**Comment by JeroenDM** [comment]
Ping @davetcoleman (I'm not sure whether I'm using "Ping" in the right context :) but the requested change is added.)

**Comment by davetcoleman** [comment]
Thanks!

**Pull request merged**